login.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Wolox Login</title>
+  <meta name="description" content="training, login card">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body class="body--white">
+  <div class="container">
+    <div class="login__wrapper">
+      <div class="login__card">
+        <div class="login__logo"></div>
+        <form id="loginForm" action="./list.html">
+          <label class="login__label" for="email">Email</label>
```

Comment:
it's better if you use more generic classes for forms. Example: `form__label`, `form__input`.